### DISCUSSION TITLE
How to use the latest version?

### DISCUSSION THREAD

**Discussion by eddynaka** [microsoft/coyote#389] [Q&A] [answered]
Hi,

I recently saw that the older packages of coyote got deprecated.
I was using something like this:
```csharp
public void AnalyzeCommandBase_ShouldGenerateSameResultsWhenRunningSingleAndMultithreaded_CoyoteTest()
        {
            Configuration config = Configuration.Create().WithTestingIterations(100).WithConcurrencyFuzzingEnabled();
            var engine = TestingEngine.Create(config, AnalyzeCommandBase_ShouldGenerateSameResultsWhenRunningSingleAndMultiThread_CoyoteHelper);
            string TestLogDirectory = ".";

            engine.Run();
            TestReport report = engine.TestReport;

            var filenames = new List<string>(engine.TryEmitTraces(TestLogDirectory, "AnalyzeCommandBase_ShouldGenerateSameResultsWhenRunningSingleAndMultiThread_CoyoteTest_Log"));
            foreach (string item in filenames)
            {
                Output.WriteLine("See log file: {0}", item);
            }

            Assert.True(report.NumOfFoundBugs == 0, $"Coyote found {report.NumOfFoundBugs} bug(s).");
        }
```

The issues:
- WithConcurrencyFuzzingEnabled got removed and I don't see any documentation on what to replace it with.
- TryEmitTraces got removed and I don't see any documentation on what to replace it with.

**Comment by eddynaka** [comment]
I was using the version 1.3.1

**Comment by pdeligia** [comment, marked as answer]
Thanks @eddynaka, these were indeed renamed/refactored in recent versions, you can replace them as follows:

- `WithConcurrencyFuzzingEnabled` is now called `WithSystematicFuzzingEnabled`.
- `TryEmitTraces` is now called `TryEmitReports`.

Hope this helps!

**Comment by pdeligia** [comment]
Closing for now, but if you have more questions please open an issue.
